test/models/ProgramLocation

```diff
@@ -0,0 +1,36 @@
+const assert = require('assert');
+const _ = require('lodash');
+const fs = require('fs-extra');
+const path = require('path');
+const { v4: uuid } = require('uuid');
+
+const helper = require('../helper');
+const models = require('../../models');
+
+describe('models.ProgramLocation', () => {
+  beforeEach(async () => {
+    await helper.loadFixtures(['programlocation']);
+  });
+
+  it('create a new ProgramLocation record', async () => {
+    let ProgramLocation = models.ProgramLocation.build({
+      Name: 'San Jose',
+      Created: '2019',
+      Pictures: 'xxx',
+      Audio: 'xxx',
```

Comment:
These attributes- Name/Created/Pictures/Audio- are not part of the ProgramLocation model.